### DISCUSSION TITLE
Copilot in VS... broke down?

### DISCUSSION THREAD

**Discussion by NotroDev** [community/community#47148] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi. I've been using copilot since beta testing.
So far everything was fine, the copilot helped me a lot, and I was happy. However, recently, it just suddenly "broke". What are the symptoms of this? The copilot icon when generating responses doesn't spin (as it used to do before), instead some new tasks appear while generating in the lower left corner where Visual Studio tasks are displayed. On top of that, the quality of the responses has gone down SIGNIFICANTLY, it generates terrible stupidity or nothing at all, so it's not useful for me.

I don't know why, it just happened suddenly.
I updated VS and reinstalled copilot 2 times, same issue.
***
Microsoft Visual Studio Community 2022
Version 17.4.4

GitHub Copilot   1.71.1.0 (v1.71.1.0@5acf71ea1)
GitHub Copilot Agent   1.71.8269 (v1.71.0)

**Comment by tilmann1** [comment]
@NotroDev Apologies for the issues, you are definitely on the latest stable version of VS and the Copilot VS extension so that's good. A few questions:

- Are you using any VS extensions in addition to Copilot?
- Which programming language are you using Copilot with?
- Do you have a reproducible scenario that GitHub engineers could try to reproduce your issues

Thanks in advance!

**Reply by NotroDev** [reply]
> * Are you using any VS extensions in addition to Copilot?

Yes. But I disabled all and it's the same.

> * Which programming language are you using Copilot with?

C#

> * Do you have a reproducible scenario that GitHub engineers could try to reproduce your issues

I don't think I can describe the entire scenario. One day it just happened (maybe some auto copilot update).

**Reply by tilmann1** [reply]
@NotroDev Can you please double check that you have all the IntelliCode options enabled? It sounds like you're only getting IntelliCode completions. To make sure you're getting Copilot completions please double check that you’ve enabled the **Show whole line completions** option in IntelliCode. Thanks!

**Reply by NotroDev** [reply]
> @NotroDev Can you please double check that you have all the IntelliCode options enabled? It sounds like you're only getting IntelliCode completions. To make sure you're getting Copilot completions please double check that you’ve enabled the **Show whole line completions** option in IntelliCode. Thanks!

Yes, I have it enabled.

**Reply by tilmann1** [reply]
@NotroDev Can you please open a support ticket via https://support.github.com/contact/bug-report and use it to share a copy of the following directory with GitHub Support: %temp%\VSFeedbackIntelliCodeLogs ? Thanks!

**Reply by NotroDev** [reply]
> @NotroDev Can you please open a support ticket via https://support.github.com/contact/bug-report and use it to share a copy of the following directory with GitHub Support: %temp%\VSFeedbackIntelliCodeLogs ? Thanks!

Done

**Comment by NotroDev** [comment]
Hey. I've updated Copilot to 1.73.2.0, same issue.

**Comment by NotroDev** [comment]
The issue still exists. I update Copilot whenever possible. I recently reinstalled Visual Studio. Copilot just gets in the way by giving me slow, very stupid, and out of context answers. In VSC and IntelliJ everything works perfectly.

**Reply by Truonghau83** [reply]
> The issue still exists. I update Copilot whenever possible. I recently reinstalled Visual Studio. Copilot just gets in the way by giving me slow, very stupid, and out of context answers. In VSC and IntelliJ everything works perfectly.

**Reply by ayim** [reply]
Hey, so sorry you're continuing to run into problems.

There's a bit of an issue with auto updating from 1.73.2 ... to get the latest, you'll actually need to uninstall Copilot and reinstalling Copilot.

Once updated to Copilot v1.77, we're super interested in doing a screenshare/live debugging this because getting good repros is hard in the AI world. Please please shoot me an email at [username]@github.com and we'll get the ball rolling.
